free-programming-cheatsheets.md

```diff
@@ -23,6 +24,11 @@
 * [Bash Scripting cheatsheet](https://devhints.io/bash) Devhints (HTML)
 
 
+### C
+
+* [C](https://users.ece.utexas.edu/~adnan/c-refcard.pdf) (PDF)
```

Comment:
We prefer to use the title as defined by the resource, please don't create your own title.

```suggestion
* [C Reference Card (ANSI)](https://users.ece.utexas.edu/~adnan/c-refcard.pdf) (PDF)
```